com.github.muriloventuroso.givemelyrics.json

```diff
@@ -0,0 +1,45 @@
+{
+    "app-id": "com.github.muriloventuroso.givemelyrics",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version" : "18.08",
+    "base" : "io.elementary.BaseApp",
+    "base-version" : "juno",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "com.github.muriloventuroso.givemelyrics",
+    "build-options" : {
+        "cxxflags": "--log-session-bus"
```

Comment:
this isn't how you do it. I meant you specify it on the `flatpak run` command line.